### ISSUE TITLE
New Clarius app update in Jan/Feb timeframe has broken interface via Mobileapi

### ISSUE THREAD

**Issue by delosa** [clariusdev/mobileapi#36]
MIT Lincoln Laboratory has written an app that uses the Mobileapi interface.   A new version of the Clarius app that was pushed to the probe in the Jan/Feb time frame is now sending a message that was not set before.

The onScanAreaChanged() method is being called.   Once this occurs the onNewProcessedImage() method stops being called.

Is there a way to turn off the some feature so the onScanAreaChanged() method will not be called or is there something I should do when I receive this message to allow the app to continue.  Right now I just log that the method has been entered.

We no longer can run our app so I would appreciate any help you can give us.

Thank you,
    Nancy DeLosa
    MIT Lincoln Laboratory

**Comment by clariusk** [comment]
we can look into this, in the meantime you can obtain previous releases of the App here: https://github.com/clariusdev/mobileapi/releases

**Comment by delosa** [comment]
Thank you we well try to install an older version.

 

 

From: Kris Dickie ***@***.***> 
Sent: Thursday, May 2, 2024 12:13 PM
To: clariusdev/mobileapi ***@***.***>
Cc: DeLosa, Nancy - 0224 - MITLL ***@***.***>; Author ***@***.***>
Subject: [EXT] Re: [clariusdev/mobileapi] New Clarius app update in Jan/Feb timeframe has broken interface via Mobileapi (Issue #36)

 

we can look into this, in the meantime you can obtain previous releases of the App here: https: //github. com/clariusdev/mobileapi/releases — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because 

ZjQcmQRYFpfptBannerStart




This Message Is From an External Sender 


This message came from outside the Laboratory. 

ZjQcmQRYFpfptBannerEnd

we can look into this, in the meantime you can obtain previous releases of the App here: https://github.com/clariusdev/mobileapi/releases

**Comment by delosa** [comment]
Hi Kris,

 

    We tried loading the older version and got a message saying that we needed to update to 11.2 to connect to the scanner.

 

    Is there something else we need to do?

 

Thank you,

    Nancy 

 

From: Kris Dickie ***@***.***> 
Sent: Thursday, May 2, 2024 12:13 PM
To: clariusdev/mobileapi ***@***.***>
Cc: DeLosa, Nancy - 0224 - MITLL ***@***.***>; Author ***@***.***>
Subject: [EXT] Re: [clariusdev/mobileapi] New Clarius app update in Jan/Feb timeframe has broken interface via Mobileapi (Issue #36)

 

we can look into this, in the meantime you can obtain previous releases of the App here: https: //github. com/clariusdev/mobileapi/releases — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because 

ZjQcmQRYFpfptBannerStart




This Message Is From an External Sender 


This message came from outside the Laboratory. 

ZjQcmQRYFpfptBannerEnd

we can look into this, in the meantime you can obtain previous releases of the App here: https://github.com/clariusdev/mobileapi/releases

**Comment by delosa** [comment]
Is there a way to download an older version of the scanner to to scanner so it works with the older app?

**Comment by clariusk** [comment]
yes, all App releases will also update the scanner software: https://github.com/clariusdev/mobileapi/releases

**Comment by delosa** [comment]
Is there a special way to update the app on the phone?   I personally did not update the app, but when running the older version the use was told to update app to 11.2 to connect to the scanner.   We must be doing something incorrectly.

**Comment by clariusk** [comment]
The Android APKs can be installed manually, you should delete any previous versions of the App beforehand
